to-do.md

```diff
@@ -3,8 +3,17 @@
 https://www.foolishdeveloper.com/2022/04/eyes-follow-mouse-cursor-javascript.html
 https://codepen.io/whipcat/pen/ExKPQqZ
 
-# add credit and add my socials
+# // add credit and add my socials
```

Comment:
This file shouldnt be here... I see its in your .gitignore so I think it may have been added to that file after you had already included it in a previous commit. Would be a good idea to add a new commit removing it

The content here is good though, you should add it to the readme as future targets